Sources/PathKit.swift

```diff
@@ -586,10 +586,19 @@ extension Path : Sequence {
       self.path = path
       self.directoryEnumerator = Path.fileManager.enumerator(atPath: path.path)!
     }
+    
+    init(path: Path, options: FileManager.DirectoryEnumerationOptions) {
```

Comment:
If I understand correctly, this initialiser is never used and it's not public. Is this here by mistake?